src/autoRespawn/main.ts

```diff
@@ -381,30 +382,61 @@ async function handleMarkChangeEvent(event: MarkChangeEvent) {
       }
 
       const { position } = addedMark
+      const { coalition = addedMark.coalition } = command
+
+      let radius = SPAWNER_MINIMUM_DISPLACEMENT_METERS
+
+      if (command.radius) {
+        radius = command.radius
+      }
 
       // make sure spawner meets minimum displacement requirement
       const nearby = await nearbySpawners({
         position,
-        accuracy: SPAWNER_MINIMUM_DISPLACEMENT_METERS,
-        coalition: Coalition.COALITION_ALL, // TODO: take a red or blue word/string from parser here
+        accuracy: radius,
```

Comment:
I feel like using radius for accuracy when the caller DOES NOT specify radius leads to a readability problem here.

It's not clear when one should or should not use radius while modifying this code moving forward, or even what radius means here.

from my outside review perspective, radius means two things here:
- when you specify redius explicitly in the destroy command, it means you want to destroy things within that range
- when you don't specify it explicity in the destroy command, it means the range to search for units to destroy, but only pick one.

however, you can't get the second behavior while also specifying radius.

to improve readability, I think we should be explicit and have two logic branches here, instead of muddying the readability by re-using a the variable for two cases.

- find one spawner nearby
- find all spawners in a given range

defining "nearby" separately from the "destroy radius"

Reply:
Do you want 2 specific db functions that return: 

* the closest unit to the position with a given accuracy
* the units in the given range

for both units and spawners or how should we do this?

Reply:
I don't think we need to be that complex to address the readability concern  I have here.

I think we can resolve my concern with something as simple as the below:

points:
- we only use radius as part of choosing when to destroy one or many, if many what to destroy is still handled by the `nearby` fns accuracy
- we don't use anything named `radius` when a radius doesn't exist on the command (so there's zero confusion about what a radius is, or the intent of a radius in this context)

```
      // search for nearby units
      let nearby = await nearbySpawners({
        position,
        accuracy: command.radius || DESTROY_SINGLE_UNIT_SEARCH_RANGE,
        coalition: coalition,
      })
      
      equal(nearby.length > 0, true, 'No existing spawners found')
      
      if (typeof command.radius !== 'number') {
        nearby = nearby.slice(0, 1)
      }
      
      await Promise.all(
          nearby.map(async element => {
              const { spawnerId } = element
              await spawnerDestroyed(spawnerId) // destroy

              // remove existing spawner marker
              const existingMarker = await findSpawnerMarker(spawnerId)

              if (existingMarker) {
                await removeMapMark(existingMarker.id)
              }

              await outText(`Spawner ${spawnerId} destroyed`)
          })
        )
```

Reply:
Didn't know you could simply do `accuracy: command.radius || DESTROY_SINGLE_UNIT_SEARCH_RANGE,`
That's pretty cool.

Not sure about renaming the property either. I would just leave it like it is.